pkg/assembler/backends/arangodb/hasSBOM_test.go

```diff
@@ -159,14 +159,14 @@ var includedTestOccurrences = []testOccurrence{{
 	isOcc: includedOccurrence,
 }}
 
-var includedHasSBOM = &model.HasSBOMInputSpec{
-	URI:              "sbom_URI",
-	Algorithm:        "sbom_algorithm",
-	Digest:           "sbom_digest",
-	DownloadLocation: "sbom_download_location",
-	Origin:           "sbom_origin",
-	Collector:        "sbom_collector",
-}
+// var includedHasSBOM = &model.HasSBOMInputSpec{
```

Comment:
commented out while @jeffmendoza works on updates to the backend tests. This will fix the lint issue for the time being before I do the release.